dpgen2/superop/block.py

```diff
@@ -226,22 +240,41 @@ def _block_cl(
     )
     block_steps.add(prep_run_dp_train)
 
-    prep_run_lmp = Step(
-        name=name + "-prep-run-lmp",
-        template=prep_run_lmp_op,
-        parameters={
-            "block_id": block_steps.inputs.parameters["block_id"],
-            "lmp_config": block_steps.inputs.parameters["lmp_config"],
-            "lmp_task_grp": block_steps.inputs.parameters["lmp_task_grp"],
-        },
-        artifacts={
-            "models": prep_run_dp_train.outputs.artifacts["models"],
-        },
-        key="--".join(
-            ["%s" % block_steps.inputs.parameters["block_id"], "prep-run-lmp"]
-        ),
-    )
-    block_steps.add(prep_run_lmp)
+    if block_steps.explore_style == "lmp":
```

Comment:
I guess the two branches may be merged.  we may change the original lmp_config->expl_config and lmp_task_grp -> expl_task_grp. 
you may need to add a dummy type_map  parameter to prep_run_lmp